requirements.txt

```diff
@@ -1,3 +1,3 @@
 asgiref==3.4.1
-Django==4.0
+Django==1.11.29
```

Comment:
Curious why we need a `requirements.txt` in addition to the `pyproject.toml` + `poetry.lock`?

Reply:
We don't need it. It's leftover.